.github/workflows/release-build-sign-upload.yml

```diff
@@ -53,7 +53,7 @@ jobs:
     steps:
 
     - name: Download Signed Linux Packages
-      uses: actions/download-artifact@v2
+      uses: actions/download-artifact@v3
```

Comment:
@moleske, this looks like a major bump. Do we know if this works?

Reply:
Why wouldn't it work? They just use a newer node internally?  Don't really understand the concern in merging it